compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/MetalRedrawer.uikit.kt

```diff
@@ -145,6 +146,7 @@ internal class MetalRedrawer(
     private val metalLayer: CAMetalLayer,
     private val callbacks: MetalRedrawerCallbacks
 ) {
+    private val metalDrawableHandler = CMPMetalDrawablesHandler(metalLayer)
```

Comment:
Please add a note with a reason here too

Reply:
Got it.